mysql-test/suite/rocksdb/r/autoinc_overflows.result

```diff
@@ -0,0 +1,901 @@
+# Testing iteration 81 with increment 9 and offset 2
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=2;
+SELECT * FROM t1;
+c1
+DROP TABLE t1;
+# Testing iteration 82 with increment 9 and offset 3
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=3;
+SELECT * FROM t1;
+c1
+DROP TABLE t1;
+# Testing iteration 83 with increment 9 and offset 4
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=4;
+SELECT * FROM t1;
+c1
+18446744073709551604
+DROP TABLE t1;
+# Testing iteration 84 with increment 9 and offset 5
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=5;
+SELECT * FROM t1;
+c1
+18446744073709551605
+DROP TABLE t1;
+# Testing iteration 85 with increment 9 and offset 6
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=6;
+SELECT * FROM t1;
+c1
+18446744073709551606
+DROP TABLE t1;
+# Testing iteration 86 with increment 9 and offset 7
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=7;
+SELECT * FROM t1;
+c1
+18446744073709551607
+DROP TABLE t1;
+# Testing iteration 87 with increment 9 and offset 8
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=8;
+SELECT * FROM t1;
+c1
+18446744073709551608
+DROP TABLE t1;
+# Testing iteration 88 with increment 9 and offset 9
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=9;
+SELECT * FROM t1;
+c1
+18446744073709551609
+DROP TABLE t1;
+# Testing iteration 89 with increment 9 and offset 10
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=9, @@session.auto_increment_offset=10;
+SELECT * FROM t1;
+c1
+18446744073709551610
+DROP TABLE t1;
+# Testing iteration 90 with increment 10 and offset 1
+CREATE TABLE t1(c1 BIGINT UNSIGNED AUTO_INCREMENT, KEY (c1)) ENGINE=ROCKSDB AUTO_INCREMENT=18446744073709551604;
+SET @@session.auto_increment_increment=10, @@session.auto_increment_offset=1;
+SELECT * FROM t1;
+c1
```

Comment:
These "empty" selects were also a bit worrying so I looked into them. It looks like we actually return ER_WARN_DATA_OUT_OF_RANGE in this case, because `compute_next_insert_id` overflows. Maybe we can add a comment in the test explaining when we get that error?